build-output-api/isr-revalidation/.vercel/output/functions/index.func/index.js

```diff
@@ -0,0 +1,14 @@
+// The bypass token can be a randomly generated string of at least 32 characters.
+// This is meant to be *private* - DO NOT expose this value on the client-side.
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
```

Comment:
Doesn't seem like this is used in this file?

Reply:
Removed!